core/modules/system/css/system.css

```diff
@@ -138,7 +138,6 @@ a.tabledrag-handle-hover .handle {
 }
 div.indentation {
   float: left; /* LTR */
-  height: 1.7em;
```

Comment:
Nice 👍🏼 ...I also wondered why we are using this when I saw it in the output of menu/taxonomy reorder-able tables.